README.md

```diff
@@ -46,6 +46,8 @@ export default function Hello() {
 
 ## Docs
 
+* [Creating custom themes](src/themes/README.md)
+
 [Docs by Storybook](https://baseui.netlify.com/)
```

Comment:
Could you add the `*` here as well, so it's a part of the unordered list of docs.